CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+2.0.4
+
+* Add method context::flush() to manually flush all inserted records to disk.
+
+---
+
```

Comment:
This `CHANGELOG.md` update doesn't belong here, it should be in the `Added function context::flush()` commit.

Reply:
Done.